### ISSUE TITLE
Missing focus outline

### ISSUE THREAD

**Issue by mbenford** [mbenford/ngTagsInput#32]
The directive is missing a focus outline to clearly indicate it's the active element on the page. That is critical for keyboard users that rely on the outline to know where they are on the page.
